build/.goreleaser.yml

```diff
@@ -218,52 +218,6 @@ nfpms:
     # Priority.
     priority: extra
 
-  - id: debian-infrastructure-agent-sysv
```

Comment:
We still need this type of service manager for `wheezy`
https://github.com/newrelic/infrastructure-publish-action/blob/main/schemas/infra-agent.yml#L70

Reply:
according to this debian 8 is minimum requirement
https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent